src/components/DownloadCSV.tsx

```
@@ -0,0 +1,104 @@
+      const blob = new Blob([data], {
+        type: "text/csv;charset=utf-8;",
+      });
+      saveAs(blob, `${benefitName}-${selectedOption}.csv`);
+
+      toast({
+        title: "CSV downloaded successfully",
+        status: "success",
+        duration: 3000,
+        isClosable: true,
+      });
+    } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)